tests/gcp/installer/util/test_data/merge_from_bucket_rules.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
I don't see all the rule files being tested: cloudsql, IAP, KE, etc.  I would have much more peace of mind if they are.